Jenkinsfile

```diff
@@ -43,11 +51,32 @@ node('vetsgov-general-purpose') {
     ]]
   ]);
 
-  stage('Validate Links') {
+  stage('Check for Urgent Changes') {
     dir(CONTENT_REPO) {
       checkout scm
+      if (IS_MASTER) return
+
+      sh "git config --add remote.origin.fetch +refs/heads/master:refs/remotes/origin/master"
+      sh "git fetch --no-tags"
+
+      def changedFiles = sh(returnStdout: true, script: "git diff --name-only origin/master..origin/${env.BRANCH_NAME}")
+      def homepageChanged = changedFiles.indexOf('fragments/home') > -1
+
+      if (homepageChanged) {
+        def message = """\
+@channel \
+Pull request opened containing changes to the VA.gov homepage! \
+These changes usually contain content that is high priority, such as for a weather alert or government shutdown. \
+Please review, merge, and if necessary, deploy this change as soon as possible. \
+https://www.github.com/${GITHUB_ORG}/${CONTENT_REPO}/pulls
+"""
+
+        slackSend(message: message, channel: 'oncall', color: '#DDDD00', failOnError: false)
+      }
     }
+  }
```

Comment:
I wanted to link to the actual PR, but every time I tried [`getNumber`](http://github-api.kohsuke.org/apidocs/index.html) an error was thrown in Jenkins.